### ISSUE TITLE
Degu Sensor Unit & A6 Gateway Reconnection Issue

### ISSUE THREAD

**Issue by alanp-sporelab** [open-degu/USER_COMMUNITY#32]
# Overview
Upon A6 Gateway power cycling, Degu Sensor Unit doesn’t automatically reconnect to the gateway.

# Setup
- Degu Sensor Unit firmware is v0.9.5
- Single Degu Sensor Unit being used 
- Python code is example code for both BMP280 and battery voltage monitoring combined:
`
def main():
    path = 'thing/' + zcoap.eui64()
    reported = {'state':{'reported':{}}}

    bus = I2C(1)
    bmp = BMP280(i2c=bus)

    while True:
        addr = zcoap.gw_addr()
        port = 5683
        cli = zcoap.client((addr, port))

        reported['state']['reported']['temp'] = bmp.values[0]
        reported['state']['reported']['pres'] = bmp.values[1]
        reported['state']['reported']['battery'] = battery_voltage()

        print(ujson.dumps(reported))
        cli.request_post(path, ujson.dumps(reported))
        time.sleep(60)
        cli.close()

if __name__ == "__main__":
    main()
`

# Issue
-  If the A6 itself has been power cycled, the Degu Sensor Unit needs to be manually reset (reset switch) or power cycled before it will reconnect to the Gateway.

# Expected outcome
- Would expect the Degu Sensor Unit to automatically reconnect when the A6 is available again.

**Comment by naomitodori** [comment]
Would you run the following Degu gateway command when this issue occurs?

`# wpanctl getprop Thread:ChildTable`

If the Degu is connected to the Thread network, it will appear as follows:

```
Thread:ChildTable = [
        "E656AF728B7811A0, RLOC16:a40d, NetDataVer:85, LQIn:3, AveRssi:-71, LastRssi:-73, Timeout:240, Age:12, RxOnIdle:yes, FTD:no, SecDataReq:yes, FullNetData:no"
] 
```

**Comment by alanp-sporelab** [comment]
Yip after rebooting / power cycling, the Degu unit reconnects to the thread network, however the Degu unit stop posting to AWS (in our case) even after it has reconnected.

**Comment by naomitodori** [comment]
Sorry, this issue is the same cause as #28 .
We will fix it and apply in the future release.
https://github.com/open-degu/USER_COMMUNITY/issues/28#issuecomment-552794660

**Comment by alanp-sporelab** [comment]
Perfect, happy to hear that. Thanks!

**Comment by yoshidam-at** [comment]
This issue fixed on [v1.0.0-rc1](https://github.com/open-degu/degu/releases/tag/v1.0.0-rc1).
Please check and close it.

How to update from v0.9.x to v1.0.0-rc1, see [Degu v0.9.xからv1.0.0-rc1移行ガイド](https://open-degu.github.io/user_manual/60_version100/).
